src/components/style/interactive/button.css

```
@@ -99,32 +99,44 @@
 
 .fdsButton--outlined {
   border: 1px solid var(--button-outlined-border);
+  background-color: white;
+  color: var(--font-color-dark);
```

```suggestion
  color: var(--font-color-primary);
```

I don't think `dark` exists any more